src/Types/index.tsx

```diff
@@ -276,192 +276,79 @@ export enum InfluxColors {
-  Duplicate = 'duplicate',
-  Duplicate_New = 'duplicate-new',
-  Edit = 'edit',
-  Erlenmeyer = 'erlenmeyer',
-  Expand_New = 'expand-new',
-  ExpandA = 'expand-a',
-  ExpandB = 'expand-b',
-  Export = 'export',
-  Export_New = 'export-new',
-  Eye = 'eye',
-  Eye_New = 'eye-open-new',
-  FileSettings = 'file-settings',
-  Function = 'fn',
-  FunnelOutline = 'funnel-outline',
-  FunnelSolid = 'funnel-solid',
-  EyeClosed = 'eye-closed',
-  EyeOpen = 'eye-open',
-  GitHub = 'github',
-  Google = 'google',
-  GraphLine = 'graphline-2',
-  GraphLine_New = 'graph-line-new',
-  Group = 'group',
-  Handle_New = 'handle-new',
-  HeartSolid = 'heart-solid',
-  Heroku = 'heroku',
-  HerokuSimple = 'heroku-simple',
-  History = 'history',
-  HomeOutline_New = 'home-outline-new',
-  Import = 'import',
-  Info_New = 'info-new',
-  Ingest_New = 'ingest-new',
-  Install = 'install',
-  Layers = 'layers',
-  Lightmode_New = 'light-mode-new',
-  Link = 'link',
-  Lock = 'lock',
-  LockOutline_New = 'lock-outline-new',
-  Logout_New = 'log-out-new',
-  Maximize = 'maximize',
-  Minimize = 'minimize',
-  Moon = 'moon',
-  More = 'more',
-  Merge = 'merge',
-  NavChat = 'nav-chat',
-  OAuth = 'oauth',
-  Octagon = 'octagon',
-  Okta = 'okta',
-  Organization = 'organization',
-  Pause = 'pause',
-  Pencil = 'pencil',
-  PieChart = 'pie-chart',
-  Pin_New = 'pin-new',
-  Play = 'play',
-  PlayOutline = 'play-outline',
-  Plus = 'plus',
-  Plus_New = 'plus-new',
-  PlusSkinny = 'plus-skinny',
-  Polaroid = 'polaroid',
-  Pulse = 'pulse-c',
-  QuestionMark_New = 'question-mark-new',
-  QuestionMark_Outline = 'question-mark-outline',
-  Redo = 'redo',
-  Refresh = 'refresh',
-  Refresh_New = 'refresh-new',
-  Remove = 'remove',
-  Remove_New = 'remove-new',
-  Save = 'save',
-  Search = 'search',
-  Search_New = 'search-new',
-  Server = 'server2',
-  Share = 'share',
-  ShareSolid = 'share-solid',
-  Shield = 'shield',
-  ShieldSolid = 'shield-solid',
-  Shuffle = 'shuffle',
-  SidebarClose_New = 'sidebar-close-new',
-  SidebarOpen_New = 'sidebar-open-new',
-  Square = 'square',
-  SquareCheck = 'square-check',
-  StackSolid = 'stack-solid',
-  Star = 'star',
-  StarSmile = 'star-smile',
-  Stop = 'stop',
-  Stop_New = 'stop-new',
-  Switch_New = 'switch-new',
-  Subscribe = 'subscribe',
-  Subtract = 'subtract',
-  Sun = 'sun',
-  Table = 'table',
-  Terminal = 'terminal',
-  Text_New = 'text-new',
-  TextBlock = 'text-block',
-  Trash = 'trash',
-  Trash_New = 'trash-new',
-  Triangle = 'triangle',
-  Undo = 'undo',
-  Upgrade = 'upgrade',
-  Upload_New = 'upload-new',
-  UploadOutline = 'upload-outline',
-  User = 'user',
-  UserAdd = 'user-add',
-  UserOutline = 'user-outline',
-  UserOutline_New = 'user-outline-new',
-  UserRemove = 'user-remove',
-  UsersDuo = 'users-duo',
-  UsersTrio = 'users-trio',
-  Wand = 'wand',
-  Wood = 'wood',
-  Wrench = 'wrench',
-  WrenchNav = 'wrench-nav',
-  Zap = 'zap',
+  AddCell_New = 'AddCell_New',
```

Comment:
In the future we should come up with a mini-epic that takes away the `_New` suffix. I realize we'd have to make the changes in clockface and UI.

Reply:
Agreed. The sheer amount of changes is the reason why the names stayed mostly the same--as much as possible at least.